nodejs/awsx/examples/api/index.ts

```diff
@@ -86,4 +87,32 @@ const api = new awsx.apigateway.API("myapi", {
     requestValidator: "ALL",
 });
 
+/**
+ * Export the url of the API.
+ */
 export const url = api.url;
+
+/**
+ * Set up the Usage Plan for the API.
+ */
+const usagePlan = new aws.apigateway.UsagePlan("my-usage-plan", {
+    apiStages: [{
+        apiId: api.restAPI.id,
+        stage: api.stage.stageName,
+    }],
+});
```

Comment:
this feels like something that would be easy and convenient to create off of our awsx.apigateway.API.   Basically `const usagePlan = myApi.createUsagePlan("name")`.

Reply:
Cool I agree! Just wanted to get your 👍 before I went down this path.